### PR TITLE
Add tooltip in the Navbar

### PR DESCRIPTION
**Pull request by preciousonah**
**Description**
- Add tooltip that shows extra information when a user hovers on a link in the navbar

**Milestones**
- Add tooltip to the Navbar that displays information about the route
- Add css styling 

Here is a preview of my website below:

Image 7-21-23 at 8 20 PM

### CODE REVIEW

File: fashion-messaging-app-ui/src/components/Homepage/Homepage.jsx [hunk 1]
```diff
@@ -2,17 +2,17 @@ import Navbar from "../Navbar/Navbar";
...
-
+import { Link } from "react-router-dom";
 
 function Homepage({photos, handleSearch}) {
   return (
     <div>
         <div className="container">
-
+        <Link to="/logout">Logout</Link>
             <Navbar />
```
Comment (speltzka) [inline fashion-messaging-app-ui/src/components/Homepage/Homepage.jsx:12]: Not sure if adding Logout is related to this change

**Review by speltzka — APPROVED** [review]
I find the Tooltip UI as shown in your demo- https://www.loom.com/share/bb141e0c79c64339a9744d366efb957a to be confusing. I think you should fix it so that the other menu items don't move when you mouse over one menu item. I also think the tooltip should provide more of a description of what the page does, right now the tooltip info seems redundant (For example when you mouse over "Messages" it just says "Open Messages"

**Comment by preciousonah** [comment]
> I find the Tooltip UI as shown in your demo- https://www.loom.com/share/bb141e0c79c64339a9744d366efb957a to be confusing. I think you should fix it so that the other menu items don't move when you mouse over one menu item. I also think the tooltip should provide more of a description of what the page does, right now the tooltip info seems redundant (For example when you mouse over "Messages" it just says "Open Messages"

Okay got it! I will work on that once i'm done with my technical challenges

File: fashion-messaging-app-ui/src/components/Homepage/Homepage.jsx (hunk 1 above)
Comment (preciousonah) [reply fashion-messaging-app-ui/src/components/Homepage/Homepage.jsx:12]: > Not sure if adding Logout is related to this change
Okay, i will try not to add unimportant code in my future pull requests

**Pull request merged**